app-arch/createrepo_c/createrepo_c-0.20.1-r1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+DISTUTILS_USE_PEP517=setuptools
+inherit cmake distutils-r1
+
+DESCRIPTION="C implementation of createrepo"
+HOMEPAGE="https://github.com/rpm-software-management/createrepo_c"
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/rpm-software-management/createrepo_c.git"
+else
+	SRC_URI="https://github.com/rpm-software-management/createrepo_c/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="doc drpm legacy test"
+RESTRICT="!test? ( test )"
+
+DEPEND="app-arch/bzip2:=
+	app-arch/rpm
+	app-arch/zchunk
+	dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/libxml2
+	dev-libs/openssl:=
+	net-misc/curl
+	sys-apps/file
+	sys-libs/libmodulemd
+	sys-libs/zlib:=
+	drpm? ( app-arch/drpm )"
+
+RDEPEND="${DEPEND}
+	app-arch/xz-utils
```

Comment:
Just checking: it doesn't use liblzma, only calls `xz` at runtime (or it `dlopen()`s liblzma?)

Reply:
I will have to look into this more. My initial assumption was wrong, it is clearly using `liblzma` as an include!